tests/profiling/check-comms.py

```diff
@@ -5,8 +5,8 @@
 t = pd.HDFStore('bw.h5')
 
 result = {
-    'MPI_ACTIVATE': { 'nb': 100, 'lensum': 12000 },
```

Comment:
There is nothing that indicate, even to a seasoned contributor to parsec, that this test need to be in sync with the communication engine. Or that if the test fails one might want to quickly check what is the size of the parsec AM header.

Reply:
See 13cc263d326ab1611db78f17a9e52f0f128399fa